tools/pot/openvino/tools/pot/algorithms/quantization/fake_quantize_configuration.py

```diff
@@ -25,13 +26,16 @@
     'bits'
 ]
 
+ACTIVATION_QUANTIZATION_MODES = ['activations', 'outputs']
```

Comment:
any chance we can rename `activations` to inputs?

Reply:
Yes. Moreover, I would like to do this in the whole POT code, but changes will be massive and may affect UX (if we will agree to change it in the tool configurations too).